Engine Hacks/Skill System/Proc Skills/proc_foresight.s

```diff
@@ -0,0 +1,66 @@
+.thumb
```

Comment:
Since this file is not actually used in the build process, please remove it (and preferably version control it elsewhere).

Reply:
(Alternatively, we could rebuild the `dmp` each build, and remove that from the repo, but that's not how this buildfile project is structured. I do think that's something a bit lacking in buildfiles rn though)